Sources/App/Controllers/HomeController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright © 2017 Patrick Balestra. All rights reserved.
+//
+
+import Vapor
+import HTTP
+
+final class HomeController {
+    
+    var droplet: Droplet!
+    
+    func addRoutes(droplet: Droplet) {
+        self.droplet = droplet
+        droplet.get(handler: self.home)
+        let home = droplet.grouped("home")
+        home.get(handler: self.home)
+    }
+    
+    func home(request: Request) throws -> ResponseRepresentable {
+        // Get all categories to display in sidebar
+        let categories = try Category.all()
+        
+        let accessories: [Accessory]
+        let featuredAccessory: Accessory
+        let featuredAccessoryImage: String
+        
+        // Limit of fetched items for accessories query
+        let visibleAccessoriesLimit = 20
```

Comment:
On a second thought, since the number of accessories in each row is 3, let's set the limit to 18 so we have exactly 6 rows.